src/RichText/RichText.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import Linkify from "react-linkify";
+
+export default function RichText(props) {
+  let properties = {};
+  if (props.newTab) {
+    properties = {target: "_blank"};
+  }
+  return (<Linkify properties={properties}>
+    {props.text.split("\n").map((item, key) => <span key={key}>{item}<br /></span>)}
```

Comment:
also, how do you think about using separate `<p>` elements rather than line breaks?